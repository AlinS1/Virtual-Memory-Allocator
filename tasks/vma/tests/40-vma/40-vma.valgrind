==5701== Memcheck, a memory error detector
==5701== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==5701== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==5701== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==5701== Parent PID: 5700
==5701== 
==5701== 
==5701== HEAP SUMMARY:
==5701==     in use at exit: 0 bytes in 0 blocks
==5701==   total heap usage: 4,562 allocs, 4,562 frees, 177,052 bytes allocated
==5701== 
==5701== All heap blocks were freed -- no leaks are possible
==5701== 
==5701== For lists of detected and suppressed errors, rerun with: -s
==5701== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
