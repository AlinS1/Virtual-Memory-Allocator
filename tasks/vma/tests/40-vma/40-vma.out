Invalid address for free.
Invalid address for write.
Invalid command. Please try again.
Total memory: 0x242465 bytes
Free memory: 0x242463 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end
Invalid address for write.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Invalid address for write.
Invalid address for write.
Invalid address for write.
Invalid address for free.
Invalid address for write.
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Invalid address for write.
Invalid address for write.
Invalid address for write.
Total memory: 0x242465 bytes
Free memory: 0x242431 bytes
Number of allocated blocks: 5
Number of allocated miniblocks: 5

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 3 end

Block 4 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 4 end

Block 5 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 5 end
Invalid address for write.
Invalid address for write.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x2421AF bytes
Number of allocated blocks: 8
Number of allocated miniblocks: 8

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 3 end

Block 4 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 4 end

Block 5 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 5 end

Block 6 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 6 end

Block 7 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 7 end

Block 8 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 8 end
Invalid address for write.
Invalid address for free.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x241FEF bytes
Number of allocated blocks: 11
Number of allocated miniblocks: 11

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 6 end

Block 7 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 7 end

Block 8 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 8 end

Block 9 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 9 end

Block 10 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 10 end

Block 11 begin
Zone: 0x110059 - 0x110099
Miniblock 1:		0x110059		-		0x110099		| RW-
Block 11 end
Invalid address for write.
Invalid address for write.
Invalid address for write.
Invalid command. Please try again.
Total memory: 0x242465 bytes
Free memory: 0x241EEF bytes
Number of allocated blocks: 12
Number of allocated miniblocks: 12

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x6DC4D - 0x6DD4D
Miniblock 1:		0x6DC4D		-		0x6DD4D		| RW-
Block 6 end

Block 7 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 7 end

Block 8 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 8 end

Block 9 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 9 end

Block 10 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 10 end

Block 11 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 11 end

Block 12 begin
Zone: 0x110059 - 0x110099
Miniblock 1:		0x110059		-		0x110099		| RW-
Block 12 end
Invalid address for write.
Invalid address for free.
Invalid address for write.
Invalid address for write.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x241E9F bytes
Number of allocated blocks: 14
Number of allocated miniblocks: 14

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x6DC4D - 0x6DD4D
Miniblock 1:		0x6DC4D		-		0x6DD4D		| RW-
Block 6 end

Block 7 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 7 end

Block 8 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 8 end

Block 9 begin
Zone: 0xBC886 - 0xBC8C6
Miniblock 1:		0xBC886		-		0xBC8C6		| RW-
Block 9 end

Block 10 begin
Zone: 0xD7929 - 0xD7939
Miniblock 1:		0xD7929		-		0xD7939		| RW-
Block 10 end

Block 11 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 11 end

Block 12 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 12 end

Block 13 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 13 end

Block 14 begin
Zone: 0x110059 - 0x110099
Miniblock 1:		0x110059		-		0x110099		| RW-
Block 14 end
Invalid address for write.
Invalid address for write.
Invalid address for write.
Invalid address for free.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x241E93 bytes
Number of allocated blocks: 16
Number of allocated miniblocks: 16

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x6DC4D - 0x6DD4D
Miniblock 1:		0x6DC4D		-		0x6DD4D		| RW-
Block 6 end

Block 7 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 7 end

Block 8 begin
Zone: 0xB66EF - 0xB66F3
Miniblock 1:		0xB66EF		-		0xB66F3		| RW-
Block 8 end

Block 9 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 9 end

Block 10 begin
Zone: 0xBC886 - 0xBC8C6
Miniblock 1:		0xBC886		-		0xBC8C6		| RW-
Block 10 end

Block 11 begin
Zone: 0xD7929 - 0xD7939
Miniblock 1:		0xD7929		-		0xD7939		| RW-
Block 11 end

Block 12 begin
Zone: 0xDB5E7 - 0xDB5EF
Miniblock 1:		0xDB5E7		-		0xDB5EF		| RW-
Block 12 end

Block 13 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 13 end

Block 14 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 14 end

Block 15 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 15 end

Block 16 begin
Zone: 0x110059 - 0x110099
Miniblock 1:		0x110059		-		0x110099		| RW-
Block 16 end
Invalid address for free.
Invalid address for write.
Invalid address for free.
Invalid address for free.
Invalid address for free.
Total memory: 0x242465 bytes
Free memory: 0x241E93 bytes
Number of allocated blocks: 16
Number of allocated miniblocks: 16

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x6DC4D - 0x6DD4D
Miniblock 1:		0x6DC4D		-		0x6DD4D		| RW-
Block 6 end

Block 7 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 7 end

Block 8 begin
Zone: 0xB66EF - 0xB66F3
Miniblock 1:		0xB66EF		-		0xB66F3		| RW-
Block 8 end

Block 9 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 9 end

Block 10 begin
Zone: 0xBC886 - 0xBC8C6
Miniblock 1:		0xBC886		-		0xBC8C6		| RW-
Block 10 end

Block 11 begin
Zone: 0xD7929 - 0xD7939
Miniblock 1:		0xD7929		-		0xD7939		| RW-
Block 11 end

Block 12 begin
Zone: 0xDB5E7 - 0xDB5EF
Miniblock 1:		0xDB5E7		-		0xDB5EF		| RW-
Block 12 end

Block 13 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 13 end

Block 14 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 14 end

Block 15 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 15 end

Block 16 begin
Zone: 0x110059 - 0x110099
Miniblock 1:		0x110059		-		0x110099		| RW-
Block 16 end
Invalid address for write.
Invalid command. Please try again.
Invalid command. Please try again.
Total memory: 0x242465 bytes
Free memory: 0x241E93 bytes
Number of allocated blocks: 16
Number of allocated miniblocks: 16

Block 1 begin
Zone: 0x31E8E - 0x31E90
Miniblock 1:		0x31E8E		-		0x31E90		| RW-
Block 1 end

Block 2 begin
Zone: 0x36FE9 - 0x37009
Miniblock 1:		0x36FE9		-		0x37009		| RW-
Block 2 end

Block 3 begin
Zone: 0x50CB8 - 0x50DB8
Miniblock 1:		0x50CB8		-		0x50DB8		| RW-
Block 3 end

Block 4 begin
Zone: 0x5EFCC - 0x5F1CC
Miniblock 1:		0x5EFCC		-		0x5F1CC		| RW-
Block 4 end

Block 5 begin
Zone: 0x6DC29 - 0x6DC2B
Miniblock 1:		0x6DC29		-		0x6DC2B		| RW-
Block 5 end

Block 6 begin
Zone: 0x6DC4D - 0x6DD4D
Miniblock 1:		0x6DC4D		-		0x6DD4D		| RW-
Block 6 end

Block 7 begin
Zone: 0x9AC76 - 0x9ACF6
Miniblock 1:		0x9AC76		-		0x9ACF6		| RW-
Block 7 end

Block 8 begin
Zone: 0xB66EF - 0xB66F3
Miniblock 1:		0xB66EF		-		0xB66F3		| RW-
Block 8 end

Block 9 begin
Zone: 0xB9310 - 0xB9318
Miniblock 1:		0xB9310		-		0xB9318		| RW-
Block 9 end

Block 10 begin
Zone: 0xBC886 - 0xBC8C6
Miniblock 1:		0xBC886		-		0xBC8C6		| RW-
Block 10 end

Block 11 begin
Zone: 0xD7929 - 0xD7939
Miniblock 1:		0xD7929		-		0xD7939		| RW-
Block 11 end

Block 12 begin
Zone: 0xDB5E7 - 0xDB5EF
Miniblock 1:		0xDB5E7		-		0xDB5EF		| RW-
Block 12 end

Block 13 begin
Zone: 0xED4B4 - 0xED4BC
Miniblock 1:		0xED4B4		-		0xED4BC		| RW-
Block 13 end

Block 14 begin
Zone: 0xFA9D5 - 0xFAA55
Miniblock 1:		0xFA9D5		-		0xFAA55		| RW-
Block 14 end

Block 15 begin
Zone: 0x107301 - 0x107303
Miniblock 1:		0x107301		-		0x107303		| RW-
Block 15 end

Block 16 begin
Zone: 0x11005